lib/jxl/decode.cc

```diff
@@ -381,6 +381,7 @@ struct JxlDecoderStruct {
   std::unique_ptr<jxl::ThreadPool> thread_pool;
 
   DecoderStage stage;
+  bool codestream_finished;
```

Comment:
I don't think we need this boolean. The decoder stage could be kept at kFinished; or have an intermediate stage before kFinished if you want to allow calling ProcessInput only if new input was added. Adding this boolean basically duplicates the "stage" as now the stage is the tuple of these two values.

Reply:
Done thanks to the other comment